front/app/containers/Admin/invitations/invite/index.tsx

```diff
@@ -470,7 +648,7 @@ const Invitations = () => {
                 }}
               />
 
-              {processing && (
+              {(processing || createInviteImportId !== null) && (
```

Comment:
Fixes existing bug, where spinner only showed while counting seats but not while creating invites.